### DISCUSSION TITLE
Bad link in gridpoints.md

### DISCUSSION THREAD

**Discussion by jfbauer432** [weather-gov/api#350] [Q&A] [answered]
At the end of gridpoints.md there is a broken link.  What's there is
`https://github.com/weather-gov/api/blob/master/reporting-issues`
but the ".md" is missing.  i.e. it should be
`https://github.com/weather-gov/api/blob/master/reporting-issues.md`

**Comment by StephenClouse** [comment]
The links are not expected to work when viewed through the repo. You probably should be looking at the actual web pages.

https://weather-gov.github.io/

**Comment by StephenClouse** [comment, marked as answer]
Whoops. Meant https://weather-gov.github.io/api/
